packages/default/scss/pdf-viewer/_layout.scss

```diff
@@ -33,61 +33,201 @@
                 }
+
+                .k-highlighting {
+                    touch-action: none;
+                }
+            }
+
+            .k-annotation-layer {
+                position: absolute;
+                top: 0;
+                left: 0;
+                transform-origin: 0 0;
+
+                section {
+                    position: absolute;
+                }
+
+                .k-annotation-text-content {
+                    position: absolute;
+                    width: 100%;
+                    height: 100%;
+                    opacity: 0;
+                    color: transparent;
+                    user-select: none;
+                    pointer-events: none;
+                }
+            }
+
+            .k-annotation-editor-layer {
+                background: transparent;
+                position: absolute;
+                inset: 0;
+                transform-origin: 0 0;
+                cursor: auto;
+
+                .k-selected {
+                    z-index: 100000 !important; // stylelint-disable-line declaration-no-important
+                }
+
+                .k-highlight-editor {
+                    position: absolute;
+                    background: transparent;
+                    z-index: 1;
+                    cursor: auto;
+                    max-width: 100%;
+                    max-height: 100%;
+                    border: none;
+                    outline: none;
+                    pointer-events: none;
+                    transform-origin: 0 0;
+
+                    .k-internal {
+                        position: absolute;
+                        top: 0;
+                        left: 0;
+                        width: 100%;
+                        height: 100%;
+                        pointer-events: auto;
+                    }
+                }
+
+                .k-free-text-editor {
+                    position: absolute;
+                    background: transparent;
+                    z-index: 1;
+                    transform-origin: 0 0;
+                    cursor: text;
+                    caret-color: $kendo-pdf-viewer-free-text-annotation-text;
+                    max-width: 100%;
+                    max-height: 100%;
+                    border: solid 2px transparent;
+
+                    &.k-selected.k-draggable {
+                        cursor: move;;
+                    }
+
+                    &.k-selected .k-internal {
+                        border-radius: 0px;
+                        outline: 2px dashed $kendo-pdf-viewer-free-text-annotation-border;
+                    }
+
+                    .k-internal {
+                        background: transparent;
+                        border: none;
+                        inset: 0;
+                        overflow: visible;
+                        white-space: nowrap;
+                        user-select: none;
+                        color: $kendo-pdf-viewer-free-text-annotation-text;
```

Comment:
I've missed these guys ( here and line 224 ), but they should also go to `_theme.scss`